README.rst

```diff
@@ -55,6 +55,11 @@ Comma-separated list of files to skip. It accepts globs as well.  Examples:
 
 * to skip directories, invoke ``codespell --skip="./src/3rd-Party,./src/Test"``
 
+The ``-r`` flag can be used to change the regular expression which is used to find words. ::
+
+    codespell -r, --regex=
+
+* to restore previous behaviour of including `_` in words, invoke ``codespell --regex="[\w\-'’\`]+"``
```

Comment:
I spent quite a while getting confused about what has to be escaped: it seems like you can get away with less escaping of this option in `bash` than you can in `sh`? Not sure exactly what is best to put here 😅

Reply:
Does the `sh` escaping work in `bash` too?